Qlever docker build fails:  stxxl and CMake < 3.5 support removed

```
CMake Error at build/_deps/stxxl-src/CMakeLists.txt:18 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.
```
see also faiing actions https://github.com/ad-freiburg/qlever/actions/runs/14124334060/job/39570096758

--- a/Dockerfile
+++ b/Dockerfile
@@ -33,7 +33,7 @@ COPY CompilationInfo.cmake /qlever/
 # Don't build and run tests on ARM64, as it takes too long on GitHub actions.
 # TODO: re-enable these tests as soon as we can use a native ARM64 platform to compile the Docker container.
 WORKDIR /qlever/build/
-RUN cmake -DCMAKE_BUILD_TYPE=Release -DLOGLEVEL=INFO -DUSE_PARALLEL=true -D_NO_TIMING_TESTS=ON -GNinja ..
+RUN cmake -DCMAKE_BUILD_TYPE=Release -DLOGLEVEL=INFO -DUSE_PARALLEL=true -D_NO_TIMING_TESTS=ON -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -GNinja ..
 RUN if  [ $TARGETPLATFORM = "linux/arm64" ] ; then echo "target is ARM64, don't build tests to avoid timeout"; fi
 RUN if [ $TARGETPLATFORM = "linux/arm64" ] ; then cmake --build . --target IndexBuilderMain ServerMain; else cmake --build . ; fi
 RUN if [ $TARGETPLATFORM = "linux/arm64" ] ; then echo "Skipping tests for ARM64" ; else ctest --rerun-failed --output-on-failure ; fi

